ui/src/index.tsx

```diff
@@ -45,6 +45,46 @@ if (params.preset instanceof Array) {
   presets = [params.preset];
 }
 
+if (params.annotateCitations !== undefined) {
+  const element = document.createElement("style");
+  document.head.appendChild(element);
+
+  if (element.sheet) {
+    element.sheet.insertRule(`
+        .citation-annotation { 
+            background-color: rgba(0, 0, 255, 0.1) !important; 
+        }`,
+      0);
```

Comment:
Oh wow! That's some cowboy engineering, if I've ever seen it!

![tenor-220678235](https://user-images.githubusercontent.com/39743357/119707345-e7f03c80-be0f-11eb-8007-11f92631c10d.gif)